website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4594,7 +4596,8 @@ modifying this behavior.
 
 The `AfterProcessingTime` trigger operates on *processing time*. For example,
 the <span class="language-java">`AfterProcessingTime.pastFirstElementInPane()`</span>
-<span class="language-py">`AfterProcessingTime`</span> trigger emits a window
+<span class="language-py">`AfterProcessingTime`</span>
+<span class="language-go">`Trigger{Kind: AfterProcessingTimeTrigger, Delay: 5000}`</span> trigger emits a window
```

Comment:
can probably simplify this down to `window.AfterProcessingTimeTrigger` which matches how users will see it. We don't need to show the wrapping, that's what the code sample blocks are for.